src/aap_eda/api/serializers/eda_credential.py

```diff
@@ -207,8 +211,9 @@ def get_references(eda_credential: models.EdaCredential) -> list[dict]:
             "type": "DecisionEnvironment",
             "id": decision_environment.id,
             "name": decision_environment.name,
-            "url": (
-                f"api/eda/v1/decision-environments/{decision_environment.id}/"
+            "uri": reverse(
```

Comment:
is this field used in some any other place? When I suggested a rename because "url" would not be accurate I didn't notice that we are already using the same struct. This would be a change of the interface and we should make sure first it doesn't brake anything else. Take into account too that the previous value was without leading slash api/eda/v1/decision-environments/{decision_environment.id}/ but reverse returns it with a leading slash
/api/eda/v1/decision-environments/[pk]/

Reply:
As far as I know, the `refs` are only used by the UI to display them to end users when they try to delete the credentials. @lgalis can you double-check? thanks

Reply:
@hsong-rh - I could not find a place where this url field is used in the ui.